R/datamapping-xrange.R

```diff
@@ -0,0 +1,33 @@
+#' @title XRangeDataMapping
+#' @docType class
+#' @description Abstract class for X Range Mapping
+#' @export
+XRangeDataMapping <- R6::R6Class(
+  "XRangeDataMapping",
+  inherit = XYDataMapping,
+  public = list(
+    yMin = NULL,
+    yMax = NULL,
+
+    # Example of how to do some other stuff

```

Comment:
Please ensure that comments are actually useful. Add a TODO if there is more to do. This can easily stay as is forever